<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NV14 Updater</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget {
background-image: url(:/resources/resources/bg.png);
}
</string>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QSplitter::handle {
    background: black;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLineEdit" name="uid1">
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>20</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#uid1 {
color: rgb(255, 255, 255);
background-color: rgb(26, 26, 26);
font: 75 20pt &quot;Courier New&quot;;
}</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string comment="CPU UID1"/>
     </property>
    </widget>
    <widget class="QLineEdit" name="uid2">
     <property name="styleSheet">
      <string notr="true">#uid2 {
color: rgb(255, 255, 255);
background-color: rgb(26, 26, 26);
font: 75 20pt &quot;Courier New&quot;;
}</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string comment="CPU UID1"/>
     </property>
    </widget>
    <widget class="QLineEdit" name="uid3">
     <property name="styleSheet">
      <string notr="true">#uid3 {
color: rgb(255, 255, 255);
background-color: rgb(26, 26, 26);
font: 75 20pt &quot;Courier New&quot;;
}</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string comment="CPU UID1"/>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="checkForUpdates">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 88, 204);
color: rgb(255, 255, 255);
font: 75 12pt &quot;Courier New&quot;;</string>
    </property>
    <property name="text">
     <string>Check for updates</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/reload.png</normaloff>:/resources/reload.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="fwList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="uidLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>NV14 Unique ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHelp">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/resources/help.png</pixmap>
    </property>
   </widget>
   <widget class="QTextBrowser" name="fwInfo">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
